rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -190,7 +195,7 @@ def test_exec(config):
                     # uploading data
                     log.info("s3 objects to create: %s" % config.objects_count)
                     if utils.check_dbr_support():
-                        if bucket_name_to_create == "brownfield-bucket":
+                        if bucket_name_to_create == brownfield_bkt_name:
```

Comment:
we can do `if bucket_name_to_create in ["brownfield-dynamic-bkt", "brownfield-manual-bkt"]` instead of assigning the value to a variable.